packages/pigeon/test/objc_generator_test.dart

```diff
@@ -415,7 +415,7 @@ void main() {
     expect(
         code,
         contains(
-            'pigeonResult.nested = [Input fromMap:GetNullableObject(dict, @"nested")];'));
+            'pigeonResult.nested = (GetNullableObject(dict, @"nested") ? [Input fromMap:GetNullableObject(dict, @"nested")] : nil);'));
     expect(code, matches('[self.nested toMap].*@"nested"'));
```

Comment:
This should really have an integration test too; there's nothing here that would fall if someone regressed the use case while making some other unrelated change to the details of this generated code (and this changed this test code too).

Reply:
Done